src/main/java/com/minecolonies/coremod/items/ItemSupplyChestDeployer.java

```diff
@@ -138,12 +138,15 @@ public EnumActionResult onItemUse(
      */
     private static boolean isFirstPlacing(@NotNull final EntityPlayer player)
     {
-        if (Configurations.gameplay.allowInfiniteSupplyChests || !player.hasAchievement(ModAchievements.achievementGetSupply))
+        /*
+        if (Configurations.allowInfiniteSupplyChests || !player.hasAchievement(ModAchievements.achievementGetSupply))
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)